### DISCUSSION TITLE
Horizontal scroll with location indicators?

### DISCUSSION THREAD

**Discussion by nicolaidanekov** [microsoft/fluentui-blazor#548] [Q&A] [answered]
Hey there!

We're using the Horizontal Scroll much like a carousel to scroll through files and other items in our Blazor app.
Is there any way to have indicators below the scroll area to show the user where they are located in the scrollable area? 
To give an idea of what I mean, here is a link to the Bootstrap Carousel with indicators:
https://getbootstrap.com/docs/4.0/components/carousel/#with-indicators

The indicators could be lines, like in the bootstrap example, or circles or whatever. :)

It is entirely this is already possible and I'm just missing a parameter, but I haven't been able to find anything regarding this myself. :) 

Thanks in advance!

**Comment by vnbaaij** [comment, marked as answer]
There is no way now to do that, but I think it would  be a good addition. I'll put it on the ToDo-list.

**Reply by nicolaidanekov** [reply]
That sounds great! :) 

Any (rough) estimate on when this might be added? :p

**Reply by vnbaaij** [reply]
Ehh..nope

**Reply by vnbaaij** [reply]
We need to examine first if the underlying web component has a notion of a 'current' item. If that is not the case, it might actually be very hard to implement this feature

**Reply by nicolaidanekov** [reply]
Yeah, guessing it might not be super easy. :D
Hope I'm wrong!
Thanks for your quick replies. 👍
